DNN Platform/Website/Providers/DataProviders/SqlDataProvider/09.09.01.SqlDataProvider

```diff
@@ -0,0 +1,15 @@
+/************************************************************/
+/*****              SqlDataProvider                     *****/
+/*****                                                  *****/
+/*****                                                  *****/
+/***** Note: To manually execute this script you must   *****/
+/*****       perform a search and replace operation     *****/
+/*****       for {databaseOwner} and {objectQualifier}  *****/
+/*****                                                  *****/
+/************************************************************/
+
+-- Add CSS contnet type, this will help when a Page Style Sheet is uploaded to a folder provider such as AzureFolderProvider.
+IF NOT EXISTS (SELECT * FROM {databaseOwner}{objectQualifier}Lists WHERE ListName='ContentTypes' AND Value = 'css')
+BEGIN
+    INSERT INTO {databaseOwner}[{objectQualifier}Lists] ( [ListName], [Value], [Text], [ParentID], [Level], [SortOrder], [DefinitionID], [Description], [PortalID], [SystemList], [CreatedByUserID], [CreatedOnDate], [LastModifiedByUserID], [LastModifiedOnDate]) VALUES (N'ContentTypes', N'css', N'text/css', 0, 0, 0, -1, N'', -1, 1, -1, '2020-02-17 09:13:51.647', -1, '2020-02-17 09:13:51.647')
```

Comment:
Was there are reason to use a hardcoded date? I see that the previous script did that, too:
https://github.com/dnnsoftware/Dnn.Platform/blob/714db32390f800ea2336bb0fd2d54fc589256e6b/DNN%20Platform/Website/Providers/DataProviders/SqlDataProvider/07.03.02.SqlDataProvider#L117-L167

Even more weird is that when I look at the `ContentTypes` list in a site I manage, all of the `CreatedOnDate` fields are set to `2000-01-01T00:00:00` and the last modified fields are `NULL`…

Any @dnnsoftware/approvers know what the deal is?  I can't imagine it makes much difference, but it certainly is confusing.

```suggestion
    INSERT INTO {databaseOwner}[{objectQualifier}Lists] ( [ListName], [Value], [Text], [ParentID], [Level], [SortOrder], [DefinitionID], [Description], [PortalID], [SystemList], [CreatedByUserID], [CreatedOnDate], [LastModifiedByUserID], [LastModifiedOnDate]) VALUES (N'ContentTypes', N'css', N'text/css', 0, 0, 0, -1, N'', -1, 1, -1, GETDATE(), -1, GETDATE())
```

Reply:
Yes, I was just following what was done before, no issue changing it to getdate()